Source/MethodCall.cs

```diff
@@ -151,39 +150,29 @@ public MethodCall(Mock mock, Condition condition, Expression originalExpression,
 			this.SetFileInfo();
 		}
 
-		public string FailMessage { get; set; }
-
-		public bool IsConditional
+		public string FailMessage
 		{
-			get { return condition != null; }
+			get => this.failMessage;
+			set => this.failMessage = value;
 		}
 
-		public bool IsVerifiable { get; set; }
+		bool IProxyCall.IsConditional => this.condition != null;
 
-		public bool Invoked { get; set; }
+		bool IProxyCall.IsVerifiable => this.verifiable;
 
-		// Where the setup was performed.
-		public MethodInfo Method { get; private set; }
-		public string FileName { get; private set; }
-		public int FileLine { get; private set; }
-		public MethodBase TestMethod { get; private set; }
+		bool IProxyCall.Invoked => this.callCount > 0;
 
-		public Expression SetupExpression
-		{
-			get { return this.originalExpression; }
-		}
-
-		public int CallCount { get; private set; }
+		MethodInfo IProxyCall.Method => this.method;
```

Comment:
Note that most properties required by `IProxyCall` are now explicitly implemented to emphasize that they're not actually needed internally.